### ISSUE TITLE
Document the doc string concepts in Reference

### ISSUE THREAD

**Issue by callendorph** [StanzaOrg/stanzasite#7]
Currently I think there is syntax like: 

```
doc:  \<doc>


   Markdown Content here 

<doc>
```

That defines a method of defining documentation related to a function or other object. This is not discussed in the reference from what I can tell.

**Comment by CuppoJava** [comment]
Agreed, the document generation system hasn't been finished, and the original plan was to report how to use this system once it's "released".
